client/packages/requisitions/src/RequestRequisition/api/operations.generated.ts

```diff
@@ -2,16 +2,16 @@ import * as Types from '@openmsupply-client/common';
 
 import { GraphQLClient, RequestOptions } from 'graphql-request';
 import gql from 'graphql-tag';
-import { ReasonOptionRowFragmentDoc } from '../../../../system/src/ReasonOption/api/operations.generated';
-import { NameRowFragmentDoc } from '../../../../system/src/Name/api/operations.generated';
+import { ReasonOptionRowFragmentDoc } from '../../../../../packages/system/src/ReasonOption/api/operations.generated';
+import { NameRowFragmentDoc } from '../../../../../packages/system/src/Name/api/operations.generated';
```

Comment:
Fix the paths?

Reply:
thanks. the gql generation threw a few curve balls.. it's still not totally reliable